pkg/network/node/vnids_test.go

```diff
@@ -146,24 +146,6 @@ func TestNodeVNIDMap(t *testing.T) {
 
 	vmap.setVNID("dup_ns1", 10, false)
 	vmap.setVNID("dup_ns2", 20, false)
-
-	checkForDuplicateNetID(t, vmap, "ns1", 10) // duplicate with dup_ns1
-	checkForDuplicateNetID(t, vmap, "ns2", 20) // duplicate with dup_ns2
```

Comment:
i think these unit tests are still helpful; you can just pass `NewNetworkPolicyPlugin()` to `newNodeVNIDMap()`

Reply:
and even do another test that you don't get an error with `NewMultiTenantPlugin()`

Reply:
Thank u I wasn't happy removing them either